Source/Lib/Codec/EbPictureDecisionProcess.c

```diff
@@ -1641,27 +1642,26 @@ void* picture_decision_kernel(void *input_ptr)
            release_prev_picture_from_reorder_queue(
                encode_context_ptr);
 
-            // If the Intra period length is 0, then introduce an intra for every picture
-            if(sequence_control_set_ptr->intra_period == 0) {
-                picture_control_set_ptr->cra_flag = EB_TRUE;
-            }
-            // If an #intra_period has passed since the last Intra, then introduce a CRA or IDR based on Intra Refresh type
-            else if(sequence_control_set_ptr->intra_period != -1) {
-                picture_control_set_ptr->cra_flag =
-                    (sequence_control_set_ptr->intra_refresh_type != CRA_REFRESH) ?
-                        picture_control_set_ptr->cra_flag :
-                        (encode_context_ptr->intra_period_position == (uint32_t) sequence_control_set_ptr->intra_period) ?
-                            EB_TRUE :
-                            picture_control_set_ptr->cra_flag;
-
-                picture_control_set_ptr->idr_flag =
-                    (sequence_control_set_ptr->intra_refresh_type != IDR_REFRESH) ?
-                        picture_control_set_ptr->idr_flag :
-                        (encode_context_ptr->intra_period_position == (uint32_t) sequence_control_set_ptr->intra_period) ?
-                            EB_TRUE :
-                            picture_control_set_ptr->idr_flag;
-
-            }
+           picture_control_set_ptr->cra_flag = EB_FALSE;
+           picture_control_set_ptr->idr_flag = EB_FALSE;
```

Comment:
For HEVC, we have a recent fix removing below two lines.
           picture_control_set_ptr->cra_flag = EB_FALSE;
           picture_control_set_ptr->idr_flag = EB_FALSE;
Could you check code on current tip of HEVC?

Reply:
The current tip of HEVC does have these removed and it will be OK to remove these in this PR.

Reply:
Hi Mark, these two lines are still here? Could you remove them?